src/sql/workbench/services/backupRestoreUrlBrowser/browser/urlBrowserDialog.ts

```diff
@@ -402,17 +346,6 @@ export class BackupRestoreUrlBrowserDialog extends Modal {
 		this.hide(hideReason);
 	}
 
-	private async generateSharedAccessSignature() {
-		this.spinner = true;
-		const blobContainerUri = `https://${this._storageAccountSelectorBox.value}.blob${this._selectedAccount.properties.providerSettings.settings.azureStorageResource.endpointSuffix}/${this._blobContainerSelectorBox.value}`;
-		const getStorageAccountAccessKeyResult = await this._azureAccountService.getStorageAccountAccessKey(this._selectedAccount, this._selectedSubscription, this._selectedStorageAccount);
-		const key1 = getStorageAccountAccessKeyResult.keyName1;
-		const createSasResult = await this._blobService.createSas(this._ownerUri, blobContainerUri, key1, this._selectedStorageAccount.name, nextYear());
```

Comment:
If we do end up being fine to remove this, then we should probably just delete the entire blob service stuff completely, it doesn't seem like anything else uses it.

https://github.com/Microsoft/azuredatastudio/blob/824196876507416ed30e6bf504cdad4de431c0f5/src/sql/workbench/services/azureBlob/browser/azureBlobService.ts